string uuid
string label
---
mesh_msgs/MeshFaceCluster cluster
